oeps/oep-38-Data-Modeling.rst

```diff
@@ -0,0 +1,442 @@
+.. vim: syntax=rst
+
+===============================
+
+OEP-38:
+
+===============================
+
++---------------+------------------------------------------------------------+
+
+| OEP | :doc:` |
+
++---------------+------------------------------------------------------------+
+
+| Title | Data Modeling Best Practices |
+
++---------------+------------------------------------------------------------+
+
+| Last Modified | 2019-08-01 |
+
++---------------+------------------------------------------------------------+
+
+| Author | Brian Beggs <bbeggs@edx.org> |
+
++---------------+------------------------------------------------------------+
+
+| Arbiter | Albemarle |
+
++---------------+------------------------------------------------------------+
+
+| Status | Provisional |
+
++---------------+------------------------------------------------------------+
+
+| Type | Architecture |
+
++---------------+------------------------------------------------------------+
+
+| Created | 2019-05-01 |
+
++---------------+------------------------------------------------------------+
+
+| Review Period | 2019-08-01 - 2019/08/16 |
+
++---------------+------------------------------------------------------------+
+
+| Resolution | - 'Original pull request' |
+
+| | - 'Update pull request' |
+
++---------------+------------------------------------------------------------+
+
+| References | - 'OEP-002' |
+
++---------------+------------------------------------------------------------+
+
+
+.. \_Original pull request:
+
+.. \_Update pull request:
+
+.. \_OEP-002:
+https://open-edx-proposals.readthedocs.io/en/latest/oep-0002-bp-repo-metadata.html
+
+Abstract
+
+========
+
+In order to obtain the highest possible value from data collected in the
+edX ecosystem, this document attempts to provide guiding thoughts and
```

Comment:
There's a lot of cases of using "edX ecosystem" and "OpenedX ecosystem".  Are these used intentionally to define different things, or are they intended to be interchangeable?   If the latter, it would be good to choose one and use it consistently.  If the former, then it probably needs a separate definition in the glossary.

Reply:
It was my intent to use openedx ecosystem. I have replaced all the references to edX ecosystem with OpenedX ecosystem..